modules/transactions/src/main/java/org/apache/ignite/internal/tx/impl/ReadWriteTransactionImpl.java

```diff
@@ -88,8 +93,12 @@ public IgniteBiTuple<ClusterNode, Long> enlist(ReplicationGroupId replicationGro
     /** {@inheritDoc} */
     @Override
     protected CompletableFuture<Void> finish(boolean commit) {
```

Comment:
It's required to update both internal and public javadocs within Transaction IgniteAbstractTransactionImpl providing the details of how finish/commit/rollback works on repeatable calls.

Reply:
I've updated java doc.